config.json

```diff
@@ -451,6 +451,13 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
+        "loops",
+        "matrices",
+        "strings",
+        "tuples",
+        "filtering",
+        "lists",
+        "variables"
 
```

Comment:
Could you please remove unnecessary space?
It's a bit annoying to see it there. Sorry to bother you with such details